pubsub/pubsub-pipe-image/pubsub-to-bigquery.py

```diff
@@ -121,7 +121,9 @@ def write_to_bq(pubsub, sub_name, bigquery):
 
 
 if __name__ == '__main__':
-    sub_name = "tweets-%s" % PROJECT_ID
+    topic_info = PUBSUB_TOPIC.split('/')
+    topic_name = topic_info[-1]
+    sub_name = "tweets%s_%s" % (topic_name, PROJECT_ID)
```

Comment:
Gotcha.  Yeah, I was just including project id for clarity, e.g. in the logs, but it's not necessary for uniqueness.
I'm happy to remove it.